src/config/translations.py

```
@@ -1,51 +1,94 @@
+    "admin_broadcast_prompt": "🔹 Введите текст рассылки:",
     "admin_delete_prompt": "Введите e-mail аккаунта для удаления:",
-    "admin_delete_success": "✅ Аккаунт успешно удален: email={email}",
-    "admin_delete_error": "❌ Не удалось удалить аккаунт: {error}",
+    "admin_delete_success": "Аккаунт удалён: email={email}",
+    "admin_delete_error": "Ошибка удаления аккаунта: {error}",
+    "reload_config_success": "Конфигурация перезагружена.",
+    "reload_config_error": "Ошибка перезагрузки конфигурации: {error}",
+    "log_not_found": "❌ Лог файл не найден",
+
+    # Аккаунты
+    "account_info": "Ваш аккаунт:\nЛогин: <code>{username}</code>\nE-mail: <code>{email}</code>\nСтатус пароля: {password_status}",
```

Ruff (0.12.2)

String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)